docs/fragments/lib/android.md

```diff
@@ -10,5 +10,4 @@ This guide shows how to build an app using our Amplify Libraries for Android and
 
 ### How are the Amplify Libraries for Android different from the AWS Mobile SDK for Android?
 
-The Amplify Android client libraries are use-case centric whereas the AWS Mobile SDK for Android is service-centric. This enables you to focus on your use-case more rather than figuring out the AWS service nuances. The Amplify libraries provide a highly abstracted category based programming model. You can also use the Mobile SDK with the Amplify libraries using escape hatches if the use case you are trying to build is not currently available in Amplify libraries.
-
+The Amplify Android client libraries are use-case centric whereas the AWS Mobile SDK for Android is service-centric. This enables you to focus on your use-case more rather than figuring out the AWS service nuances. The Amplify libraries provide a highly abstracted category based programming model. You can also use the Mobile SDK with the Amplify libraries using escape hatches if the use case you are trying to build is not currently available in Amplify libraries.
```

Comment:
Suggestions: 
"use-case more rather" -- remove the word "more"
"rather than figuring out the AWS service nuances." -- rather than figuring out low level AWS service nuances
"You can also use the Mobile SDK" - We may choose not to rely on the mobile sdk right now -- perhaps it would be better put as: "You can access lower level service clients by using escape hatches.." (this isn't technically true for auth since we are currently using AWSMobileClient.. but... hmm i think calling it lower level service clients is more accurate)

Reply:
per discussion offline, i suspect we will update this at a later time.